.github/workflows/autobump.yml

```
@@ -0,0 +1,55 @@
+name: Bump formulae on schedule or request
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/workflows/autobump.yml
+  workflow_dispatch:
+    inputs:
+      formulae:
+        description: Custom list of portable-ruby formulae to livecheck and bump if outdated
+        required: false
+  schedule:
+    # Daily at midnight
+    - cron: "0 0 * * *"
+
+env:
+  HOMEBREW_FORCE_HOMEBREW_ON_LINUX: 1
+
+permissions:
+  contents: read
+
+jobs:
+  autobump:
+    if: github.repository == 'Homebrew/homebrew-portable-ruby'
+    runs-on: ubuntu-22.04
+    container:
+      image: ghcr.io/homebrew/ubuntu22.04:master
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+        with:
+          core: false
+          cask: false
+          test-bot: false
+
+      - name: Configure Git user
+        uses: Homebrew/actions/git-user-config@master
+        with:
+          username: ${{ (github.event_name == 'workflow_dispatch' && github.actor) || 'BrewTestBot' }}
+
+      - name: Get list of autobump portable-ruby formulae
+        id: autobump
+        run: echo "autobump_list=$(xargs < "$(brew --repo homebrew/homebrew-portable-ruby)"/.github/autobump.txt)" >> "$GITHUB_OUTPUT"
```

Rather not have to duplicate names into an autobump.txt:

```suggestion
        run: echo "autobump_list=$(brew tap-info --json "${GITHUB_REPOSITORY}" | jq -r '.[].formula_names[]')" >> "$GITHUB_OUTPUT"
```

---

adopted 👍